internal/command/cliconfig/config_unix.go

```diff
@@ -13,6 +13,13 @@ import (
 	"path/filepath"
 )
 
+// Default directories to use when XDG_* is not defined
+// https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+const (
+	defaultConfigDir = ".config"
+	defaultDataDir   = ".local/share"
```

Comment:
Per https://github.com/opentofu/opentofu/issues/303#issuecomment-1906209913  I think we don't want to add the non XDG_ fallbacks at this point.

Reply:
Sure. I only touched `configDir` because it's the only function that has a side-effect when things aren't found. I reverted back to explicit XDG lookup.